packages/p2p/src/stores/general-store.js

```diff
@@ -760,12 +758,11 @@ export default class GeneralStore extends BaseStore {
             this.ws_subscriptions.order_list_subscription.unsubscribe();
             return;
         }
-
-        const { p2p_order_list, p2p_order_info } = order_response ?? {};
+        const { p2p_order_info } = order_response ?? {};
```

Comment:
doesn't this line come under setP2pOrderList function? so where do we get the value for order_response from ? i dont see setP2pOrderList being called anywhere